staff/tomas-sancheznavarro/js-arrays/every.js

```diff
@@ -0,0 +1,9 @@
+function every(arr, callback) {
+
+    for (var i = 0; i < arr.length; i++) {
+        if (!callback(arr[i])) {
```

Comment:
Maybe you can change the line to be able to send the index and the array to callback. With this modfication you would allow users to use them in the callback, according to MDN page. 

`callback(arr[i], i, arr)`

Reply:
Correction applied! Thanks!